File: 250316/시력 검사 2/eye-test-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double real_number = sc.nextDouble();

        if(real_number >= 1.0) {
            System.out.println("High");
        }
        else if(real_number >= 0.5) {
            System.out.println("Middle");
        }
        else {
            System.out.println("Low");
        }
    }
}

File: 250316/시력 검사 2/eye-test-2.cpp
#include <iostream>
using namespace std;

int main() {

    double eye = 0;
    cin >> eye;
    
    if(eye >= 1.0) {
        cout << "High";
    }
    else if(eye >= 0.5) {
        cout << "Middle";
    }
    else {
        cout << "Low";
    }
    return 0;
}